tests/sles4sap/hana.pm

```diff
@@ -0,0 +1,139 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
```

Comment:
Would you mind using a BSD(-like) license and not GPL here? E.g. the text which you also used in sles4sap/hana_test.pm or is there a specific reason why you want to use GPL here?

```
# Copying and distribution of this file, with or without modification,
# are permitted in any medium without royalty provided the copyright
# notice and this notice are preserved.  This file is offered as-is,
# without any warranty.
```

Reply:
Agree.